src/crawlee/basic_crawler/_basic_crawler.py

```diff
@@ -88,14 +88,27 @@ class BasicCrawlerOptions(TypedDict, Generic[TCrawlingContext]):
 
 
 class BasicCrawler(Generic[TCrawlingContext]):
-    """Provides a simple framework for parallel crawling of web pages.
-
-    The URLs to crawl are fed either from a static list of URLs or from a dynamic queue of URLs enabling recursive
-    crawling of websites.
-
-    `BasicCrawler` is a low-level tool that requires the user to implement the page download and data extraction
-    functionality themselves. If we want a crawler that already facilitates this functionality, we should consider using
-    one of its subclasses.
+    """A basic web crawler providing a framework for crawling websites.
```

Comment:
Any chance we could make the class names links?

Reply:
I have no idea, opened a new issue for that -> https://github.com/apify/crawlee-python/issues/625.